src/azul/service/service_config/request_config.json

```diff
@@ -53,46 +53,46 @@
   },
   "manifest": {
     "bundles": {
-      "bundle_uuid": "uuid",
```

Comment:
Unfortunately, we can't modify the bundle and file column names because that would break the CLI.